astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

```diff
@@ -600,39 +308,35 @@ In real life, these applications would have different functionality. For this gu
 * Change `index.js` constants to use the Hooli values for the title (to 'Hooli'), hostname (`hooli.local`), port (`3001`), and the Client Id and Client Secret (from the admin UI application screen).
 * Change the layout so that the menu links to the Pied Piper application. Make sure to include the port.
 
-```pug title="Hooli layout view"
-doctype html
-html
-  head
-    title= title
-    link(rel='stylesheet', href='/stylesheets/style.css')
-  body
-    h2
-      a(href='http://piedpiper.local:3000') Pied Piper
-      |
-      || Hooli
-    block content
-```
+<RemoteCode
+  url={frontmatter.codeRoot + "/hooli/views/layout.pug"}
+  lang="pug" 
+  title="Index view" />
 
 * Start the application on the port `3001`. Use a different terminal window so that you can have both Node applications running at once.
 
 ```shell script title="Starting up the Hooli application"
 PORT=3001 npm start
 ```
 
-And that's it. You've just created a second application.
+And that's it. You've just created a second application. Congrats!
 
 ## Test The Results
 
-You can visit `http://piedpiper.local:3000`. You'll be redirected to the FusionAuth login screen. Log in. You'll be greeted with a welcome message by the Pied Piper app. Click on the 'Hooli' link and you'll be automatically signed in to that application. You can log out as well.
+Here's how you can test the work you've just done:
+
+* Visit `http://piedpiper.local:3000`. You'll be redirected to the FusionAuth login screen. 
+* Check <InlineUIElement>Keep Me Signed In</InlineUIElement>
+* Log in.  You'll be greeted with a welcome message by the Pied Piper app.
+* Click on the 'Hooli' link and you'll be automatically signed in to that application. 
 
 Here's a demo video of the single sign-on process from the end user perspective:
 
 <YouTube id="UV9FW57dfbQ" />
```

Comment:
This needs a cover image in YouTube!

Reply:
I tightened the video up a bit, maybe we could move to it?

https://youtu.be/KxxLLoS74Ks

Reply:
Great, updated to use your new video.